x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/persist_rule.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { renderHook, act } from '@testing-library/react-hooks';
+
+import { usePersistRule, ReturnPersistRule } from './persist_rule';
+import { ruleMock } from './mock';
+
+jest.mock('./api');
+
+describe('usePersistRule', () => {
+  test('init', async () => {
+    const { result } = renderHook<unknown, ReturnPersistRule>(() => usePersistRule());
```

Comment:
Would it make more sense to use `void` instead of `unknown` here since the callback has no props, not unknown props?

`renderHook()` signature:

``` ts
/**
 * Renders a test component that will call the provided `callback`, including any hooks it calls, every time it renders.
 */
export function renderHook<P, R>(callback: (props: P) => R, options?: RenderHookOptions<P>): RenderHookResult<P, R>;
```